src/streamlink/plugins/twitch.py

```diff
@@ -175,12 +184,14 @@ class TwitchHLSStreamWorker(HLSStreamWorker):
     def _reload_playlist(self, text, url):
         return load_hls_playlist(text, url, parser=TwitchM3U8Parser)
 
+    def _set_playlist_reload_time(self, playlist, sequences):
+        self.playlist_reload_time = sequences[-1].segment.duration
```

Comment:
seems a bit to much for normal Streams without `low-latency`
even if Twitch is doing it in the browser